docs/manual-EN/3.build-develop-and-administration/3.deploy-and-administrations/server-administration/configuration-statements/configs-syntax.md

```diff
@@ -1,9 +1,19 @@
 # CONFIG Syntax
 
-**Nebula Graph** uses `gflags` for run-time configurations.
+## Configuration Introduction
+
+**Nebula Graph** gets configuration from meta by default. If you want to get configuration locally, please add `local_config=true` option in flag files `metad.conf`, `storaged.conf`, `graphd.conf` respectively.
```

Comment:
add the `local_config=true` option in flag files `metad.conf`, `storaged.conf`, and `graphd.conf` respectively.